tests/cacl/test_cacl_application.py

```diff
@@ -32,7 +32,9 @@ def ignore_hardcoded_cacl_rule_on_dualtor(tbinfo):
 
 
 @pytest.fixture(scope="module")
-def docker_network(duthost):
+def docker_network(duthosts, rand_one_dut_hostname):
```

Comment:
instead of rand_one_dut_hostname, could we use enum_rand_one_per_hwsku_frontend_hostname. That will make compatible with a T2 chassis as well.

Reply:
Used enum_rand_one_per_hwsku_hostname to run test on both sup and one of the Linecards.